client/elements/sc-page-search.html

```diff
@@ -211,6 +217,13 @@
                handle-as="json"
                loading="{{loadingResults}}"
                on-response="_didRespond"></iron-ajax>
+    
+    <iron-ajax
+               auto
```

Comment:
Using `auto` might slow down the render of this element until the suttaplex data is returned by the server. Please confirm that doesn't happen using the devtools performance tab.